config/database.yml

```diff
@@ -4,7 +4,7 @@ default: &default
   host: <%= ENV.fetch("PG_HOST", 'localhost') %>
   database: diaper_dev
   username: <%= ENV.fetch("PG_USERNAME", 'postgres') %>
-  password: <%= ENV.fetch("PG_PASSWORD", nil) %>
+  password: <%= ENV.fetch("PG_PASSWORD", 'root') %>
```

Comment:
Please don't change this (especially not as a side effect of a feature PR), it's the default for everyone who has the code checked out. If your own password is `root`, you need to set an environment variable `export PG_PASSWORD=root`.